package.json

```diff
@@ -19,7 +22,6 @@
     "eslint-plugin-import": "^1.12.0",
     "eslint-plugin-jsx-a11y": "^2.0.1",
     "eslint-plugin-react": "^6.0.0",
-    "react": "^15.3.0",
```

Comment:
You have to let it in `devDependencies` because `npm i` doesn't install `peerDependencies`.
Duplicate it.